tools/local-presubmit

```diff
@@ -17,14 +17,16 @@ status "Testing all of the things."
 
 set -x
 
-./tools/sigh webpack
-./tools/bazelisk ${BAZELRC_OPTS} test //java/... //javatests/... //src/... //particles/...
+#./tools/sigh webpack
```

Comment:
I guess you weren't intending to include this file?

Reply:
I intend for it to be here right now :-) But it will not be included in the final merge.
 
There were some flakey tests, so I am fixing those. This change made it easier to iterate faster.

If you want to remove your approval, I can re-request review when I am happy with test results.

Reply:
That's fine. this only disables local presubmit so you could modify it locally without committing it.

Reply:
I am committing the change to the file because I want the tests to run this way on the cloud build machine.

Reply:
Ok, I've had a good run of ~20 tests passing on GCB, so I've reverted the local-presubmit file changes, and might actually be ready to merge this PR. :-)

Reply:
sounds good. Thanks for the work here, these were a couple of really nasty bugs.